jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/WebAppContext.java

```diff
@@ -746,13 +740,13 @@ public String[] getServerClasses()
     }
 
     @Override
-    public boolean isServerClass(Class<?> clazz)
+    public boolean isHiddenClass(Class<?> clazz)
     {
         return _serverClasses.match(clazz);
```

Comment:
still calling this _serverClasses and not _hiddenClasses? (like you did in ee10?)